Sample Data/Event/QradarOffenses.json

```diff
@@ -0,0 +1,1222 @@
+[
```

Comment:
Name of the sample data file should be same as legend name which is **QRadarOffense_CL**